src/popup/components/Modals/Default.vue

```diff
@@ -62,24 +63,30 @@ export default {
 @use '../../../styles/variables';
 @use '../../../styles/mixins';
 
-.icon-box {
-  @include mixins.flex(center, center, column);
+.default {
+  .text-heading-2 {
```

Comment:
can we change `.text-heading-2` to `.heading`

Reply:
This class name is defined globally.